Data/Yaml/Config.hs

```diff
@@ -0,0 +1,136 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE OverloadedStrings #-}
+-- | Some next-gen helper functions for the scaffolding's configuration system.
```

Comment:
Comment should be updated to not reference "scaffolding", but instead give a local description of the functionality.